recipe/conda_build_config.yaml

```diff
@@ -9,17 +9,37 @@ blas_impl:
 cross_compiler_target_platform:  # [win]
   - win-64                     # [win]
 c_compiler:
-  - toolchain_c                # [unix and not ppc64le]
+  - toolchain_c                # [linux or osx]
+  # new compilers
+  - gcc                        # [linux]
+  - clang                      # [osx]
+
   - gcc                        # [ppc64le]
+
   - vs2008                     # [win]
   - vs2015                     # [win]
   - vs2015                     # [win]
 cxx_compiler:
-  - toolchain_cxx              # [unix and not ppc64le]
+  - toolchain_cxx              # [linux or osx]
+  # new compilers
+  - gxx                        # [linux]
+  - clanggxx                   # [osx]
+
   - gxx                        # [ppc64le]
+
   - vs2008                     # [win]
   - vs2015                     # [win]
   - vs2015                     # [win]
+
+# these need to be split later
+channel_sources:
+  - conda-forge,defaults
+  - conda-forge/label/new,conda-forge,defaults  # [linux or osx]
+
+channel_targets:
+  - conda-forge main
+  - conda-forge new                             # [linux or osx]
```

Comment:
I am not sure I like the name `new` for the label. It seems as though it is not descriptive enough.